content/competitions/codejam-2023.html

```diff
@@ -0,0 +1,137 @@
+            </p>
+
+            <h3>So, how does it all work?</h3>
+            <p>The CodeJam is programming competition hosted online through
+                <a href="https://www.hackerrank.com/">HackerRank</a>. You can code in virtually any language (see
+                <a href="https://www.hackerrank.com/environment">competition environment</a>).
+                Once you’ve written your code, it’s executed online against a number of test cases. </p>
+            <p>Tests will cover basic scenarios as well as larger and more complex cases to ensure your algorithm is
+                efficient and reliable. Points are awarded for the number of tests you pass and are also based on the
+                difficulty of the question.</p>
+            <p>There will be a leaderboard of competitors throughout the event for you to track number of points
+                scored. The top scorers by the end of the day, as mentioned, will earn prizes!</p>
+
+            <h3>How can I prepare?</h3>
+            <p>You'll need a HackerRank account to compete; you can sign up
+                <a href="https://www.hackerrank.com/">here</a>. You don’t need to do special practice or
+                preparation, just come well-rested and ready to problem solve! If you'd like, you can try some practice
+                questions on HackerRank to get familiar with the interface and style of questions. If you've never used
+                HackerRank before, not to worry! We will be going through a short demo of how it works at the start of
+                CodeJam. You can also sign up to the competition on HackerRank in advance of the day:
+
+            <div style="text-align: center;">
+                <a class="button is-success" href="https://hackerrank.com/uqcs-codejam-2023">Sign Up To The HackerRank Competition</a>
+            </div>
+
+            </p>
+            <p>If you are coming in person, make sure to get a ticket too!</p>
+
+            <div style="text-align: center;">
+                <a class="button is-success" href="https://events.humanitix.com/uqcs-codejam-2023">Get An In-Person Ticket</a>
+            </div>
+
+            <h3>What do I do on the day?</h3>
+            <p>The CodeJam will run from 10am – 4pm on Saturday 1st April. The live event will be hosted from the
+                Advanced Engineering Building, room 49-301, but you can participate from anywhere. However, you must be competing
+                in-person to be eligible for prizes. You can join in at any time but the more time you spend, the better your
+                chances are of winning prizes!
+            </p>
+            <p>We'll also have free lunch on the day too, so be sure not to miss out!</p>
+            <p>We’ll post updates and reminders throughout the day on our <a href="http://discord.uqcs.org/">Discord
+                    server</a>.
+                Both the event kickoff and presentation of prizes will be live streamed on our
```

Comment:
Aside: should we live stream these again this year? as there probs wont be many people competing online given the new rules

Reply:
I think they'd be good to livestream just to serve as a memento on the YouTube channel especially the prize presentations